src/netcdf_io/interp_inc.fd/driver.F90

```diff
@@ -76,12 +76,14 @@ program interp_inc
  real(8), allocatable :: rlon(:), rlat(:), crot(:), srot(:)
  real(8), allocatable :: gi(:,:), gi2(:,:), go(:,:), go2(:,:), go3(:,:)
 
-
+ logical :: jedi = .false.
+ character(len=11) :: ice_inc_name
+ 
  ! NOTE: u_inc,v_inc must be consecutive
  data records /'u_inc', 'v_inc', 'delp_inc', 'delz_inc', 'T_inc', &
                'sphum_inc', 'liq_wat_inc', 'o3mr_inc', 'icmr_inc' /
 
- namelist /setup/ lon_out, lat_out, outfile, infile, lev
+ namelist /setup/ lon_out, lat_out, outfile, infile, lev, jedi
```

Comment:
@RussTreadon-NOAA OK, I just realized I made a mistake anyway and need to revise. My motivation with the namelist parameter was to mimick what I did in https://github.com/NOAA-EMC/GSI-utils/pull/35 and https://github.com/NOAA-EMC/global-workflow/pull/2420 , but perhaps this executable is called elsewhere, not just in ush/calc_analysis.py

Reply:
While `interp_inc.x` is referenced in several g-w jobs, it looks like it is only used in the analcalc jobs.  I don't know if regional DA systems or other users of gsi_utils.fd execute `interp_inc.x`.   Adding `jedi` to the namelist works.  We just need to open more issues and PRs.

Reply:
The problem is that interp_inc.x isn't just reading a netcdf file, it's creating one too with those variable names. Because of that, there's no error to catch on this line https://github.com/NOAA-EMC/GSI-utils/blob/bb03e172e0d0d9c56d6da7788ca033bfb5ef5119/src/netcdf_io/interp_inc.fd/driver.F90#L188 The variable name is just going to be whatever you define it to be.

Reply:
OK.  A logical switch is needed.